docs/cookbooks/common_workflows.md

```diff
@@ -0,0 +1,97 @@
+# Common bigquery-etl workflows
+
+This is a quick guide of how to perform common workflows in bigquery-etl using the `bqetl` CLI.
+
+For any workflow, the bigquery-etl repositiory needs to be locally available, for example by cloning the repository, and the `bqetl` CLI needs to be installed by running `./bqetl bootstrap`.
+
+## Adding a new scheduled query
```

Comment:
We should cross-link between this section and wlach's longer-form article on adding a new scheduled query. These are good complementary views.

Reply:
As a note, the cookbook I made was based on an earlier version that @scholtzan shared with me. I do worry a bit about these going out of sync with each other -- I guess my question is, does it have value to have this short form version? The tutorial isn't *that* long.